### ISSUE TITLE
F24 failure on table examples ?

### ISSUE THREAD

**Issue by luce-carevic** [w3c/aria-practices#2376]
Not sure if it's a WCAG failure (F24) but Firefox uses my custom colors settings instead of rendering the text black although I've not overridden default color with mine.

![image](https://user-images.githubusercontent.com/6073336/172552557-c4f12dfe-cacf-422c-91c3-bdc921b46a6c.png)

The problem comes from the fact that the color of the text is set with `color: initial;` 

```
table.data td, table.data th, 
table.index td, table.index th,
table.widget-features td, table.widget-features th {
[…]
  color: initial;
}
```

Setting the rule to `color: inherit;` fixes it.

See MDN: 
> On [inherited properties](https://developer.mozilla.org/en-US/docs/Web/CSS/inheritance#inherited_properties), the initial value may be unexpected. You should consider using the [inherit](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit), [unset](https://developer.mozilla.org/en-US/docs/Web/CSS/unset), [revert](https://developer.mozilla.org/en-US/docs/Web/CSS/revert), or [revert-layer](https://developer.mozilla.org/en-US/docs/Web/CSS/revert-layer) keywords instead.
